openfisca-web-api/deprecated-api.md

```diff
@@ -0,0 +1,639 @@
+# Deprecated API
+
+> This is the documentation for the old OpenFisca web API.  
+> Note that it isn't updated and that its stability isn't guaranteed anymore.  
```

Comment:
You can still find it on `api.openfisca.fr` but note that it won't be updated and that its stability isn't guaranteed.

Reply:
Done with a link to GitHub instead as the french instance isn't for all users.